### PR TITLE
introduce `buildTargetABIs` property to realm-jni project in order to allow to build only specified ABI(s)

### PR DESCRIPTION
**Pull request by zaki50**
example(only build jni files for arm64 and x86_64):
```
./gradlew assemble -PbuildTargetABIs=arm64,x86_64
```

example(build jni files for connected device and execute connectedCheck):
```
./gradlew connectedCheck -PbuildTargetABIs=$(adb shell getprop ro.product.cpu.abi)
```

@realm/java

### CODE REVIEW

**Comment by cmelchior** [comment]
Nice idea 👍 
Perhaps we should add a section at the bottom of the README with the various configuration options we have no. There is starting to be quite a bit, and it would be nice for new hires.

**Comment by emanuelez** [comment]
Nice! 👍

**Comment by stk1m1** [comment]
Super nice! 👍 💯

**Comment by zaki50** [comment]
added a check of ABI names from `buildTargetABIs` property and small example to `README.md`.

please review last 2 commits. @realm/java

File: realm/realm-jni/build.gradle [hunk 1]
```diff
@@ -83,7 +83,21 @@ def allTargets = [
     new Target( name:'x86_64',  abi:'x86_64',      toolchain:toolchains.find {it.name == 'x86_64'}, cflags:[] )
 ]
 
-def targets = ext.buildTargetAbis == null ? allTargets : allTargets.findAll {ext.buildTargetAbis.contains(it.abi)}
+def targets
+if (ext.buildTargetAbis == null) {
+    targets = allTargets;
+} else {
+    targets = ext.buildTargetAbis.collect {
+        def targetAbi = it
```
Comment (emanuelez) [inline realm/realm-jni/build.gradle]: why introducing this variable? if you want `it` to be renamed you can just use `collect { targetAbi -> ... }`

Comment (zaki50) [inline realm/realm-jni/build.gradle]: Right!

fixed.

**Comment by emanuelez** [comment]
:shipit:

**Comment by cmelchior** [comment]
:+1:

**Pull request merged**